README.md

```diff
@@ -3,4 +3,17 @@
 
 # Penumbra
 
-Penumbra is a free and open source library for GPU accelerated solar shading calculations using [pixel counting](http://www.ibpsa.org/proceedings/BS2011/P_1271.pdf).
+Penumbra is a free and open source library for GPU accelerated solar shading calculations using [pixel counting](http://www.ibpsa.org/proceedings/BS2011/P_1271.pdf). It is configured as a cross-platform CMake project. 
+
+## Pre-requisites:
+
+1. A C++ compiler (e.g., Clang, GCC, MSVC)
+2. CMake
+
+## Building penumbra from source
+
+1. Clone the git repository, or download and extract the source code (`tar.gz` or `zip`).
```

Comment:
Can you check to see if the compressed source folders contain the submodule code? If not, then we'll have to modify these instructions.

Reply:
It would be really cool if every time a new Penumbra release comes out it triggers a GitHub Action where it updates the link to the latest `tar.gz` and `zip` file in the README.

Reply:
Did you check if the compressed source folders contain the submodule code?